### DISCUSSION TITLE
Keybind Placeholder Expansion

### DISCUSSION THREAD

**Discussion by EllairaX** [PlaceholderAPI/PlaceholderAPI#582] [(3) Ideas]
### Type

- [X] New function for PlaceholderAPI.  
A new function that developers could use.
- [ ] Change to code (Internal).  
Changes to code that won't affect the end-user.
- [ ] Change to code (External).  
Changes to code that will affect the end-user (breaks stuff).
- [ ] Other: __________ 

### Description
I would like to suggest an expansion with adds placeholders for user keybinds, in case someone has a different sprint key, crouch key, etc. Unfortunately I am not experienced enough to suggest how this may be done. I mentioned this idea in the Discord server and was told to post it here by a staff member.

**Comment by Andre601** [comment]
I can imagine this to be doable, but probs requires some work, especially for all the possible keybinds.

I myself will look into this and see what can be made.

**Comment by Andre601** [comment]
After a conversation with devs of libraries such as Kyori's Adventure-library is it apparently not possible to retrieve the keybind used by the player... At least not in a very basic way. It would require PAPI to send the response or similar in a completely different way, which isn't doable as of now and probably never will with v3 being in the works.

I'm sorry that I can't give you a better response to this. Maybe in the future will there be a solution, but for now is that a not doable thing.

**Reply by EllairaX** [reply]
Thank you, I appreciate it!
